website/project/model.py

```diff
@@ -2291,6 +2292,15 @@ def update_node_wiki(self, name, content, auth):
         )
         new_page.save()
 
+        updated_page = dict(
+            page_name=name,
+            version=version,
+            user=auth.user,
+            node=self
+        )
+
+        wiki_updated.send(updated_page)
```

Comment:
These can be implemented as positional or named arguments. Consider doing 
```
wiki_updated.send(
    page_name=name,
    version=version,
    user=auth.user,
    node=self,
)
```
also modifying the signature of subscribe_wiki_updates to reflect this.